### PR TITLE
Dapperize the etcd image for v3

### PR DESCRIPTION
**Pull request by galal-hussein**
@LLParse Can you please review

### CODE REVIEW

File: Dockerfile.dapper [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+ENV DOCKER_URL_amd64=https://get.docker.com/builds/Linux/x86_64/docker-1.10.3 \
+    DOCKER_URL_arm=https://github.com/rancher/docker/releases/download/v1.10.3-ros1/docker-1.10.3_arm \
+    DOCKER_URL=DOCKER_URL_${ARCH}
+
+RUN wget -O - ${!DOCKER_URL} > /usr/bin/docker && chmod +x /usr/bin/docker
+
+ENV DAPPER_SOURCE /go/src/github.com/rancher/etcd-catalog/
+ENV DAPPER_OUTPUT ./bin ./dist
+ENV DAPPER_DOCKER_SOCKET true
+ENV TRASH_CACHE ${DAPPER_SOURCE}/.trash-cache
```
Comment (LLParse) [inline Dockerfile.dapper]: Can we add a line `ENV DAPPER_ENV REPO` in here, so the host's `REPO` environment variable may be passed to dapper during local development phase?

**Comment by LLParse** [comment]
Worked like a charm, thanks!

Along with this PR, could we please delete the entire contents of the `compose-raw`, `images` and `templates` folders? We no longer need it on master.

File: Dockerfile.dapper (hunk 1 above)
Comment (galal-hussein) [reply Dockerfile.dapper]: sure i will add that

**Comment by LLParse** [comment]
LGTM

**Pull request merged**